test/support/commands.js

```diff
@@ -66,6 +66,8 @@ Cypress.Commands.overwrite('visit', (originalFn, url = '/', options = {}) => {
 
   let waits = [
     '@routeWorkspaceClinicians',
+    '@routeDirectories',
```

Comment:
routeDirectories probably always should have been here.. these are the requests for when a workspace changes.. _we could_ make this just `routeWorkspaceClinicians` as well since it's currently last.. adding directories meant I needed to change one test from `wait` to `get`